test/unit/Libify.js

```diff
@@ -0,0 +1,308 @@
+            type: 'credit_alphanum12',
+            code: 'MONSTER',
+            issuer: 'GCOXOGREKTFJJJP2RTZHRHQC6IB7AHPSBQGUAVIMDMTQNJZSNLCJLT2G'
+          },
+          sendMax: '5',
+          path: [
+            {
+              type: 'native'
+            }
+          ],
+          destAsset: {
+            type: 'native'
+          },
+          destAmount: '456789'
+        }
+      },
+      {
+        id: 3,
+        name: 'manageOffer',
+        attributes: {
+          selling: {
+            type: 'native'
+          },
+          buying: {
+            type: 'credit_alphanum12',
+            code: 'ALLTHETHINGS',
+            issuer: 'GCVUWFL4USUCGHCPCOLPMORYCI5F37IM3CEIQM55IQ3ZNDSYCQIE2534'
+          },
+          amount: '0',
+          price: '4.417',
+          offerId: '0'
+        }
+      },
+      {
+        id: 4,
+        name: 'createPassiveOffer',
+        attributes: {
+          selling: {
+            type: 'native'
+          },
+          buying: {
+            type: 'credit_alphanum4',
+            code: 'GOOD',
+            issuer: 'GCVUWFL4USUCGHCPCOLPMORYCI5F37IM3CEIQM55IQ3ZNDSYCQIE2534'
+          },
+          amount: '5',
+          price: '2'
+        }
+      },
+      {
+        id: 5,
+        name: 'setOptions',
+        attributes: {
+          inflationDest: 'GAJNYCPCVHNM724OKKJ3UOPLLHP7LUEODUMXLE4RZ74OXAYENY5BV4CZ',
+          setFlags: 1,
+          clearFlags: 6,
+          masterWeight: '1',
+          lowThreshold: '2',
+          medThreshold: '3',
+          signerPubKey: 'GA2IKCQR3WNN5W5446MU5UZZW7UIHWQN6UVJCAXRMRH4JV3QSOABQXI4',
+          signerWeight: '5',
+          homeDomain: 'example.com'
+        }
+      },
+      {
+        id: 6,
+        name: 'changeTrust',
+        attributes: {
+          asset: {
+            type: 'credit_alphanum4',
+            code: '123',
+            issuer: 'GB6I3Y3F24UMOJ326W3Z4AHRI3VKXNEFSDPZ7CCSHHYD5ML7RDEJYHJR'
+          }
+        }
+      },
+      {
+        id: 7,
+        name: 'allowTrust',
+        attributes: {
+          trustor: 'GB6I3Y3F24UMOJ326W3Z4AHRI3VKXNEFSDPZ7CCSHHYD5ML7RDEJYHJR',
+          assetCode: '456',
+          authorize: 'true'
+        }
+      },
+      {
+        id: 8,
+        name: 'accountMerge',
+        attributes: {
+          destination: 'GD4WIAWVK7TCHDLIVB7YQ3P45PGPPK4RD52FZVS52TJE3224R566G45L',
+          sourceAccount: 'GDNG66PN7A4OCNY2XOAWL4NVO6ALFRP2RCGOZ2WMGMHBZCWM5RXWVHPU'
+        }
+      },
+      {
+        id: 9,
+        name: 'inflation',
+        attributes: {}
+      },
+      {
+        id: 10,
+        name: 'manageData',
+        attributes: {
+          name: 'wow',
+          value: 'such test'
+        }
+      }
+    ]);
+    let decodedTx = xdr.TransactionEnvelope.fromXDR(largeTransaction.xdr, 'base64');
+    let opAtIndex = (index) => {
+      return Operation.operationToObject(decodedTx._attributes.tx._attributes.operations[index]);
+    };
+
+    describe('1st operation', () => {
```

Comment:
Using ordinals here for your description is a bit cumbersome... I would prefer you either use the operation name, the index of the operation, or add comments in the structure above that specifies the ordinal of each operation.